src/CodeGenerator/ConfigProcessor.php

```diff
@@ -76,37 +76,37 @@ public static function addRules(Config $config, array $specialClassesAndProperti
         $hasPropertyAssembler = new OpenEuropa\Assembler\HasPropertyAssembler();
 
         return $config
-        //            // Add the ResultInterface to classes that match given regex.
-        //            ->addRule(
-        //                new Rules\TypenameMatchesRule(
-        //                    new Rules\AssembleRule(new Assembler\ResultAssembler()),
-        //                    '/Response$/'
-        //                )
-        //            )
-        // Set the default property assembler and generate all properties.
+            //            // Add the ResultInterface to classes that match given regex.
+            //            ->addRule(
+            //                new Rules\TypenameMatchesRule(
+            //                    new Rules\AssembleRule(new Assembler\ResultAssembler()),
+            //                    '/Response$/'
+            //                )
+            //            )
```

Comment:
I see this repo is full of commented out code, not for this PR but I suggest you make a ticket/issue where you uncomment this code and either move it to a separate branch, or add a flag to execute it only then true, but leaving code like this is a nightmare

Reply:
I will clean all of this in a follow up ticket, before closing the epic and releasing.